tests/DocBlock/TypeExpressionTest.php

```diff
@@ -651,20 +651,42 @@ public static function provideAllowsNullCases(): iterable
 
     public function testWalkTypes(): void
     {
-        $typeExpression = new TypeExpression('Foo|Bar|Baz', null, []);
-        $addLeadingSlash = static function (TypeExpression $type): void {
+        $typeExpression = new TypeExpression('Foo|Bar|($v is \Closure(X, Y): Z ? U : (V&W))', null, []);
+
+        $addLeadingSlashFx = static function (TypeExpression $type): void {
```

Comment:
Personally I don't like the `fx` convention. What is the meaning of that?

Reply:
callable variable - `$addLeadingSlash` could be otherwise read as a flag, `Fx` suffix is good coding convention for more self explanatory code

Reply:
We have multiple closures like this in the `src` and there is no such convention, and I am not sure we should introduce it. Let's make those variables compatible with rest of the codebase (drop `Fx` suffix). I am not totally against this idea, but I like standardisation, so if we want such suffix, let's enforce it and change all the occurences 🙂.

Reply:
In `master` there are only to `Fx` usages, in `TypeExpressionTest::testSortTypes()`, done by you in the past. Since this is out of the scope of this PR, we can keep them, but let's not introduce 2 new occurrences.

Reply:
done